modules/database/src/ioc/db/dbChannel.h

```diff
@@ -41,20 +41,30 @@ extern "C" {
 /**
  * event subscription
  */
-typedef struct evSubscrip {
+struct evSubscrip;
+
+typedef struct evSubscrip evSubscrip;
+
+#ifdef EPICS_PRIVATE_API
+struct evSubscrip {
     ELLNODE             node;
     struct dbChannel  * chan;
+    /* user_sub==NULL used to indicate db_cancel_event() */
     EVENTFUNC         * user_sub;
     void              * user_arg;
+    /* associated queue, may be shared with other evSubscrip */
     struct event_que  * ev_que;
+    /* NULL if !npend.  if npend!=0, pointer to last event added to event_que::valque */
     db_field_log     ** pLastLog;
     unsigned long       npend;      /**< n times this event is on the queue */
     unsigned long       nreplace;   /**< n times replacing event on the queue */
+    /* DBE mask */
```

Comment:
If docygen markup is a question of all or nothing, then I am inclined towards nothing.  I could also move this definition to some *Pvt.h if you would prefer.

Reply:
Since they won't get processed they don't need to be Doxygen markup as such, it's just harder to read data member comments that are above or below the member instead of on the same line, my brain always goes "is this comment about the one above or below?".

Reply:
Ok, I understand a desire for consistency.  I have move the remaining suffix comments to the previous line.